source/remediation_runbooks/scripts/test/test_enable_delivery_status_logging.py

```diff
@@ -62,8 +76,20 @@ def test_enables_delivery_status_logging(mocker):
         event = { 'topic_arn': topic_arn, 'logging_role': logging_arn, 'sample_rate': '0' }
         response = lambda_handler(event, {})
         assert response == { 
-            "FailureFeedbackRole": logging_arn,
-            "SuccessFeedbackRole": logging_arn,
-            "SuccessSampleRate": logging_arn }
+            "LambdaFailureFeedbackRoleArn": logging_arn,
+            "LambdaSuccessFeedbackRoleArn": logging_arn,
+            "LambdaSuccessFeedbackSampleRate": logging_arn,
```

Comment:
Should the expected sample rate really be the logging arn?

Reply:
No and it scares me that it just passes like that...